cookbook/routing/index.rst

```diff
@@ -9,6 +9,8 @@ Routing
     redirect_in_config
     method_parameters
     service_container_parameters
-    custom_route_loader
     redirect_trailing_slash
     extra_information
+    extending_routing
+        custom_route_loader
+        multiple_routers
```

Comment:
not sure if its ok to add multiple levels of navigation here, but i think it would make sense. also not sure if this is the right syntax for rst.